ingestion_server/ingestion_server/api.py

```diff
@@ -163,6 +200,12 @@ def on_post(self, req, res):
                 "task_id": task_id,
                 "status_check": status_url,
             }
+        elif is_bad_request.value:  # set to 1 for bad tasks
+            res.status = falcon.HTTP_400
+            res.media = {
+                "message": "Failed during task execution due to bad request. Check "
+                "scheduler logs."
```

Comment:
Just a nit. I'm surprised the linter allows this form actually, it's quite unreadable in my opinion :confused: 

```suggestion
                "message": (
                  "Failed during task execution due to bad request. Check "
                  "scheduler logs."
                )
```

Reply:
Fixed, along with the other such string in d220079.